### PR TITLE
docs: update publishing instructions

### PR DESCRIPTION
**Pull request by villebro**
📜 Documentation

Update the release instructions to be more in line with regular usage.

### CODE REVIEW

**Review by zhaoyongjie — APPROVED** [review]
nice!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,5 +64,8 @@ committing. You can use `npm run commit` to help you create a commit message.
 **Prerequisite:** You'll need an [npmjs.com](https://npmjs.com) account that is part of the
 `superset-ui` organization.
 
-1. Make sure you're logged in to NPM from your shell. Run `npm login` if necessary.
-2. To make the release, run `yarn run release` and follow the prompts.
+1. Checkout the `master` branch from the main repo at `apache-superset/superset-ui` - NOT A FORK!
+2. run `yarn run ci:create-patch-version` to bump the patch version (the most common case) or
+   `yarn run ci:create-minor-version` to bump the minor version. Once the process finishes and the
+   commit has been pushed to GitHub, CI will complete publishing the release to npm (takes some time
+   to complete).
```
Comment (eschutho) [inline CONTRIBUTING.md:71]: this is great. Do you want to explain that this pushes the changes directly to master? Is there a dry run option?

Comment (eschutho) [reply CONTRIBUTING.md:71]: also you need to be a committer on superset-ui. Aaron added me to the organization but I was unable to push to master.. are there other permissions that need to be granted to push directly to master?

Comment (villebro) [reply CONTRIBUTING.md:71]: No dry run option that I'm aware of. Thanks for catching the part about needing to be a committer - I updated the prerequisite to mention that you need to be a committer on `superset-ui` 👍  Let me check if you're missing some rights to the repo.

**Pull request merged**